apps/dashboard/app/models/user_configuration.rb

```diff
@@ -83,22 +89,6 @@ def navbar_type
     end
   end
 
-  # What to group pinned apps by
-  # @return [String, ""] Defaults to ""
-  def pinned_apps_group_by
```

Comment:
We can't remove this because this is how the `pinned_apps` landing page can be grouped. You can see the group by section here in the documentation.


https://osc.github.io/ood-documentation/latest/customization.html#pinning-applications-to-the-dashboard

Reply:
I didn't remove it, I updated the code to support 2 new properties in `FeaturedApp` (feature and sub_feature).
UserConfiguration still has `pinned_apps_group_by`:

```
# What to group pinned apps by
# @return [String, ""] Defaults to "
ConfigurationProperty.property(name: :pinned_apps_group_by, read_from_env: true, default_value: ""),
```

By adding the 2 new properties to `FeaturedApp`, there is no need to do the `original_category`, `original_subcategory` logic. When creating a FeaturedApp from a pinned_app, the code will set feature to and sub_feature to . This will have the same effect as before.

The reason for this change is simply to remove the logic for `original_category` and  `original_subcategory`.

What do you think?

Reply:
Sorry, I didn't notice that it had gotten moved. Moving it is fine, I just didn't want to _remove_ it entirely.